src/Administration/Resources/app/administration/src/module/sw-sales-channel/page/sw-sales-channel-create/index.js

```diff
@@ -31,29 +31,34 @@ export default {
     },
 
     methods: {
-        async createdComponent() {
+        createdComponent() {
             if (!this.$route.params.typeId) {
                 return;
             }
 
-            const measurementUnits = await this.getMeasurementUnits();
-
             if (!Shopware.Store.get('context').isSystemDefaultLanguage) {
                 Shopware.Store.get('context').resetLanguageToDefault();
             }
 
             this.salesChannel = this.salesChannelRepository.create();
             this.salesChannel.typeId = this.$route.params.typeId;
             this.salesChannel.active = false;
+
+            this.setMeasurementUnits().then(() => {
```

Comment:
```suggestion
            this.setMeasurementUnits().finally(() => {
```

Some error notification would be nice, but at least the page should remain functional if this req fails.

Reply:
Derp, you're totally right.